src/database/queries/reviews.js

```diff
@@ -14,19 +14,27 @@ exports.findAll = async (filter, attributes = {}) => {
 		return error
 	}
 }
-exports.countDistinct = async (filter, attributes = {}) => {
+
+exports.countDistinct = async (filter, attributes = ['resource_id']) => {
```

Comment:
@adithyadinesh0412 why attribute are hardcorded here, pass from the function instaed of doing that